ispdb/yahoo.com

```diff
@@ -14,7 +14,6 @@
     <domain>yahoo.com.mx</domain>
     <domain>ymail.com</domain>
     <domain>rocketmail.com</domain>
-    <domain>yahoodns.net</domain>
```

Comment:
I removed this as yahoodns.net is used for aol and yahoo. AOL accounts are defaulting to yahoo as provider and those will fail to authenticate.

Reply:
Have you actually tried that? Because if the @ email domain is in the ISPDB directly, then the DNS MX lookup won't be used. The yahoodns.net probably helps with lots of other domains that are not explicitly listed. So, I don't think it's a good idea to remove that.

Reply:
We tried adding compuserve.com/netscape.com/cs.com accounts to Thunderbird. They all ended up trying to use yahoo.com as oauth provider and that will never work. You have to be connecting to aol.com for these aol accounts to get oauth tokens and access as well. I am checking with person who requested netscape.com to find out whether netscape.com were for employees or users.

We also locally tested that after the change, the oauth access worked and were able to provision the account on Thunderbird.

I can add mx-aol.mail.gm0.yahoodns.net and mx.gm0.yahoodns.net. Using yahoodns.net is bad as Thunderbird is routing both aol & yahoo accounts to yahoo. Let me know.

Reply:
> We tried adding compuserve.com/netscape.com/cs.com accounts to Thunderbird. They all ended up trying to use yahoo.com as oauth provider and that will never work

Did you try before or after adding compuserve.com to these files? You can test them with files on disk. Details on https://developer.mozilla.org/en/Thunderbird/Autoconfiguration , search for "harddisk". What I'm saying is that adding the domain explicitly to aol.com config will take precedence over `yahoodns.net` for Yahoo config.

> You have to be connecting to aol.com for these aol accounts to get oauth tokens and access as well

Ah, yes, [AOL OAuth2 servers are known](https://searchfox.org/comm-central/source/mailnews/base/src/OAuth2Providers.jsm#31). If you use the AOL config file and `imap.aol.com`, you will get the AOL OAuth2 server.

Reply:
> Using yahoodns.net is bad

yahoodns.net should stay as fallback for all other Yahoo domains. There are lots, and removing that will break them.

As mentioned, having a more specific domain for AOL should take precedence, so it should work.

You can test the specific domain lookup with the harddisk config, but not the DNS config.

Reply:
I reverted the yahoodns.net change back. For now leaving this alone and it will broken for aol domains that are not present in configuration files.